arch/risc-v/src/mpfs/Kconfig

```diff
@@ -247,6 +247,16 @@ config MPFS_DMA
 
 menu "MPFS Others"
 
-endmenu
+config MPFS_USE_MMU_AS_MPU
```

Comment:
I would say that this should be the default, and only supported behavior for MPFS. To simplify, Perhaps just remove the flag and always use MMU for protected build?

Reply:
No problem